src/AbstractFixer.php

```diff
@@ -63,6 +63,10 @@ public function __construct()
 
     final public function fix(\SplFileInfo $file, Tokens $tokens)
     {
+        if ($this instanceof ConfigurableFixerInterface && null === $this->configuration) {
```

Comment:
Because the `null !== $this->configuration` check was missing so you could run the fixer without calling `configure()` first. Now you must call it, which will then give the configuration to `OptionsResolver` which will then validate it.